### ISSUE TITLE
Building Project Error For Arm V7

### ISSUE THREAD

**Issue by careylipu** [polyfloyd/trollibox#18]
gitpod /workspace/trollibox/bin (master) $ export GOOS=linux
gitpod /workspace/trollibox/bin (master) $ export GOARCH=arm
gitpod /workspace/trollibox/bin (master) $ export GOARM=7
gitpod /workspace/trollibox/bin (master) $ RELEASE=1 ./build.sh

error:

rm: cannot remove '/workspace/trollibox/src/handler/webui/static/css/app.css': No such file or directory
*** Building Project ***
fork/exec /tmp/go-build4277199085/b001/exe/minify: exec format error

**Comment by polyfloyd** [comment]
Yeah, it's probably compiling the minify build tool with the same arch settings but it fails because it has to run on the host.

I'll see what I can do about this...
